apps/frontend/src/pages/onboarding/styles.module.css

```diff
@@ -62,9 +71,15 @@
 		font-size: 28px;
 		line-height: 30px;
 	}
+}
 
-	.button-container {
-		flex-direction: column;
-		max-width: 250px;
+@media (width <= 480px) {
+	.answers {
+		width: 100%;
+	}
+
+	.question {
+		font-size: 24px;
+		line-height: 26px;
```

Comment:
According to figma design for mobile screen font-size for question should be 20px and line-height 25px.